src/notation/internal/notationinteraction.cpp

```diff
@@ -1570,7 +1570,7 @@ bool NotationInteraction::doDropStandard()
 //! NOTE: Helper method for NotationInteraction::drop. Handles drop logic for text base items & symbols (returns "accepted")
 bool NotationInteraction::doDropTextBaseAndSymbols(const PointF& pos, bool applyUserOffset)
 {
-    EngravingItem* el = elementAt(pos);
+    EngravingItem* el = m_dropData.dropTarget ? const_cast<EngravingItem*>(m_dropData.dropTarget) : elementAt(pos);
```

Comment:
Let's avoid the `const_cast` here and just drop the `const` qualifier from `m_dropData.dropTarget`. Some other methods will need to be tweaked as a result, but it shouldn't be anything too crazy.

Apart from this everything looks good to me, thanks!

Reply:
That actually entails quite a lot a changes...
If it's the const_cast< > that bothers you we could avoid it just by walking the list of elements on the page and finding the one that has the same pointer value as m_dataData.dropTarge (which is essentially what elementAt( ) does, except it's by cartesian position).

Reply:
The linked changes seem sensible to me. I have some future changes in mind for the drag/drop interaction which would involve making the drop target non-const anyway.

Reply:
Ok, I pushed that as a separate commit then - do I need to uncheck the "no unnecessary changes" checkbox? :-P